src/config/consts.js

```diff
@@ -2,6 +2,42 @@ const ONCE_A_DAY = 'ONCE_A_DAY';
 const ONCE_PER_HOUR = 'ONCE_PER_HOUR';
 
 export const consts = {
+  a11yLabel: {
+    address: 'Adresse',
+    backIcon: 'Zurück Taste',
+    backIconHint: 'Navigieren zurück zur vorherigen Seite',
+    bookmarkList: 'Lesezeichenliste (Taste)',
+    bookmarkListHint: 'Zu der Lesezeichenliste hinzufügen',
+    button: 'Taste',
+    category: 'Kategorie',
+    closeMenuIcon: 'Schließen Taste',
```

Comment:
(Taste) needs brackets

Reply:
solved in 62c6f5c318a48af248f0b6bc0380987d343bbce5